plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -72,7 +72,7 @@ class StaticHtmlReporter : Reporter {
     private val licensesSha1 = mutableMapOf<String, String>()
 
     override fun generateReport(input: ReporterInput, outputDir: File, config: PluginConfiguration): List<File> {
-        val reportTableModel = ReportTableModelMapper.map(input)
+        val reportTableModel = TableReportModelMapper.map(input)
```

Comment:
Commit message:

> Rename the model to `TableReport`

"... model and mapper ..."

> table based and consist

"table-based and consist*s*"

Reply:
If you want to emphasize the multiple tables, just swapping "ReportTable" to "TableReport" does not add much clarity IMO. I'd rather go for plural "tables" and use "ReportTables" or "TablesReport" (personally, I'd have a preference for the former as it more resembles "IssueTable" etc.).

Reply:
The variable `reportTableModel` here isn't renamed accordingly, although the function parameter below *are* renamed.